Dangerfile

```diff
@@ -0,0 +1,28 @@
+has_app_changes = !git.modified_files.grep(/lib/).empty?
+has_test_changes = !git.modified_files.grep(/spec/).empty?
+
+# Sometimes it's a README fix, or something like that - which isn't relevant for
+# including in a project's CHANGELOG for example
+declared_trivial = github.pr_title.include? '#trivial'
+
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn('PR is classed as Work in Progress') if github.pr_title.include? '[WIP]'
+
+# Warn when there is a big PR
+warn('Big PR') if git.lines_of_code > 500
+
+# Add a CHANGELOG entry for app changes
+if !git.modified_files.include?('CHANGELOG.md') && has_app_changes
+  fail('Please include a CHANGELOG entry.')
+end
+
+# Warn about un-updated tests
+if has_app_changes && !has_test_changes
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Favor modifier if usage when having a single-line body. Another good alternative is the usage of control flow &&/||.